Main/ClientApp/src/Components/CollectionsSite.tsx

```diff
@@ -13,7 +13,12 @@ export default function CollectionsSite(props: CollectionsSiteProps) {
         <Route
           exact
           path={path}
-          component={() => <Message text="Select a collection to display" />}
+          component={() => (
+            <PanelWideMessage
+              text="Select a collection to display"
+              throbber={false}
```

Comment:
I'd give the throbber a default value of false so we don't have to specify it everywhere